src/flask_config.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+
+class FlaskConfig:
+    def __init__(self):
+        self.secret_key = os.environ.get('FLASK_SECRET_KEY').strip()
+        self.login_disabled = os.environ.get('FLASK_LOGIN_DISABLED') or False
```

Comment:
You can provide a default result directly here as a second argument, e.g. `os.environ.get('FLASK_LOGIN_DISABLED', False)`